src/webserver.go

```diff
@@ -179,6 +179,16 @@ func initDBconnection() {
 	}
 }
 
+func TeslaMateAPIHandleErrorResponse(c *gin.Context, s string) {
+    log.Println("[error] TeslaMateAPICarsChargesDetailsV1 " + c.Request.RequestURI + " error in execution! " + s)
```

Comment:
```suggestion
    log.Println("[error] " + s1 + " - " + c.Request.RequestURI + " error in execution! " + s2)
```

Reply:
Great catch, completely missed this. Wasn't paying enough attention 😅